docs/examples/multiple-agent-shared-memory.md

```diff
@@ -0,0 +1,93 @@
+This example shows how to use one `Agent` to load content into `TaskMemory` and get that content from another `Agent` using `VectorStoreClient`.
+
+The first `Agent` uses a remote vector store (`MongoDbAtlasVectorStoreDriver` in this example) to handle memory operations. The second `Agent` uses the `VectorStoreClient` with the same `MongoDbAtlasVectorStoreDriver` to get the data.
+
+The `MongoDbAtlasVectorStoreDriver` assumes that you have a vector index configured where the path to the content is called `vector`, and the number of dimensions set on the index is `1536`.
```

Comment:
Why 1536? Is this a hardcoded magic # or something customers will have to adjust based on their vector store?

Reply:
yes this is apparently the magic number that is used for the `text-embedding-ada-002` model. not sure about others.

Reply:
Most embedding models generate vectors of size 1536. Haven't come across one that generates of any other dimension.

Reply:
It probably belongs on the Embedding Drivers page. We can link to it from here if we want.